lib/features/mailbox/data/network/mailbox_api.dart

```diff
@@ -401,6 +405,52 @@ class MailboxAPI with HandleSetErrorMixin {
     return listMailboxIdSubscribe ?? [];
   }
 
+  List<String> updateRightsForSubaddressing(MailboxSubaddressingAction action, List<String>? currentRights) {
+    const String postingRight = 'p';
```

Comment:
please define `const` in global

Reply:
both `postingRight` and `anyoneIdentifier` are needed by two files:
- `lib/features/mailbox/data/network/mailbox_api.dart`
- `model/lib/extensions/presentation_mailbox_extension.dart`

should i create some sort of `constants.dart` file to define them ?